### DISCUSSION TITLE
What is the status of this repo

### DISCUSSION THREAD

**Discussion by 1nky** [GimelStudio/GimelStudio#201] [General]
Hi Love the idea will happily contribute.... I would like to know if it is still being actively developed the last code entry is dated over a year ago?
thanks

**Comment by Correct-Syntax** [comment]
Hello @1nky, welcome!  

As for the status of the project, yes, we (the maintainers) haven't had the time to give to the project the past few years but we still have the desire to bring it to life. We are "back to the drawing board", re-thinking our approach and what direction Gimel Studio is headed before proceeding.

The existing Python code is largely a POC and will probably not be developed further. 

If you would like to see an idea I was working on earlier this year, see [here](https://github.com/Correct-Syntax/gimel_studio_ui_test). More to come... 👀 

Not sure if you joined our chat [here](https://gimelstudio.zulipchat.com/join/sif32f3gjpnikveonzgc7zhw/) already, but your contributions to the direction would be more than welcome.

What aspect of what is listed in the README most interests you? What use-case would you use Gimel Studio for? 

The core idea of non-destructive image editing is a good concept, but we want to make Gimel Studio useful and that requires knowing what sort of workflows Gimel Studio should cater towards.
